trestle/common/list_utils.py

```diff
@@ -79,7 +79,33 @@ def delete_item_from_list(item_list: List[Any], value: Any, key: Callable[[Any],
     return item_list
 
 
+def get_item_from_list(
+    item_list: Optional[List[Any]], value: Any, key: Callable[[Any], Any], remove: bool = False
+) -> Any:
+    """Get item from list if present."""
+    if not item_list:
+        return None
+    keys = [key(item) for item in item_list]
+    item = None
+    if value in keys:
```

Comment:
What if there are multiple values in the array? Should we return multiple values then? If not specify in the doc that only the first one will be returned

Reply:
Docs now emphasize first